src/geometry/angle.rs

```diff
@@ -312,18 +311,18 @@ impl Neg for Angle {
     }
 }
 
-impl ApproxEq for Angle {
```

Comment:
The docs for this trait also caused problems with the linkchecker. We don't use `approx_eq` anywhere else than in the test, so I decided to just move the trait into the test module.